debian/rules

```diff
@@ -3,6 +3,7 @@ export DH_VERBOSE=1
 
 include /usr/share/dpkg/pkg-info.mk
 FLAKE8 := $(shell flake8 --version 2> /dev/null)
+DIST_RELEASE := $(shell lsb_release -cs)
```

Comment:
This needs an extra build-depends. Can't we parse `/etc/os-release`? It's not identical in its structure in all releases, though, one would have to check all the way back from trusty.

Reply:
Can do this instead with the following (relying on our own code which is tested on each release we support)
https://paste.ubuntu.com/p/srMkS7DgR5/

Reply:
updated this to use VERSION_ID from /etc/os-release as that avoids python in debian/rules and is an available field on all supported series.